components/atoms/PrimaryCTAButton.tsx

```diff
@@ -20,11 +21,16 @@ interface PrimaryCTAButtonProps {
   text: string;
   color: string;
   backgroundColor: string;
+  linkHref?: string;
 }
 
-const PrimaryCTAButton = ({ text, color, backgroundColor }: PrimaryCTAButtonProps) => (
+const PrimaryCTAButton = ({
+  text, color, backgroundColor, linkHref,
+}: PrimaryCTAButtonProps) => (
   <PrimaryCTAButtonStyled backgroundColor={backgroundColor}>
-    <DefaultParagraph color={color}>{text}</DefaultParagraph>
+    <DefaultParagraph color={color}>
+      {linkHref && <PsychoLink href={linkHref}>{text}</PsychoLink>}
```

Comment:
Does this mean that without linkHref it does not really make sense to use this component?

Reply:
Since PsychoLink is an extension of the Next.js Link component, and the latter requires this property, it means that yes

Reply:
Then I don't think you need `linkHref && <Component />`, just make the property required ?

Reply:
hmm fair enough